API/Tensorflow/native_libs/src/c_wrappers/tensors.cpp

```diff
@@ -7,6 +7,7 @@
 #include <vector>
 #include <memory>
 #include <random>
+#include <fstream>
 
 template<typename T> T* vector_as_array(const std::vector<T>& vec) {
```

Comment:
Not a part of this pull request, but this function should be in an anonymous namespace.